.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Configen CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
```

Comment:
Is this true?

Reply:
I forgot to remove the default template comments. I will update now.